.aspectplugins

```diff
@@ -2,3 +2,8 @@
 #- name: hello-world
 #  version: v0.2.0
 #  from: github.com/aspect-build/aspect-cli-plugin-template
+
+# Example: Aspect's pro plugin
+# - name: plugin-aspect-pro
+#   from: http://static.aspect.build
```

Comment:
```suggestion
#   from: https://static.aspect.build
```

Reply:
I actually think we should throw a warning message if it's http. It's super easy to impersonate a domain and inject malicious code.

Reply:
We don't have SSL setup for the S3 bucket. Adding one looks quite non-trivial, as with anything AWS there are several other services to configure perhaps? Or we could have cloudflare as the SSL termination point and have it proxy the traffic but of course that adds another point of failure and latency

Reply:
> The reason for all of this seemingly unnecessary complexity is that CloudFront can handle SSL connections with a custom domain, but S3 cannot. (Perhaps, someday, AWS will get around to building a simpler solution and this entire post will feel quaint.)

Reply:
Yeah, let's do this in another PR.